lib/rucio/client/metaconventionsclient.py

```diff
@@ -21,8 +21,11 @@
 from rucio.client.baseclient import BaseClient
 from rucio.client.baseclient import choice
 from rucio.common.utils import build_url
-from rucio.db.sqla.constants import KeyType
-from typing import Union, Optional
+from typing import Union, Optional, TYPE_CHECKING
```

Comment:
The `isort`-compliant order would be:

```suggestion
from typing import TYPE_CHECKING, Optional, Union
```

but I think we can ignore it here. I have opened a PR that also adds an `isort` check in the CI: https://github.com/rucio/rucio/pull/6578, so this will be fixed later on